Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -756,15 +756,15 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::Load(const u32 stage)
   }
 
   // how many levels the allocated texture shall have
-  const u32 texLevels = hires_tex ? (u32)hires_tex->m_levels.size() : tex_levels;
+  const u32 texLevels = custom_tex ? (u32)custom_tex->m_levels.size() : tex_levels;
```

Comment:
This PR is actually my first ever contribution to any such sproject in the opensource git/github arena, I'm also not a real programmer and only know C++ basics, this change I believe would have to be consulted with others if they are okay with it being part of this PR, but then the Title and the PR message would not be valid anymore.